rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -449,3 +450,19 @@ def test_force_delete_used_blueprints_via_import(self):
                           first_blueprint_id)
 
         self.client.blueprints.delete(first_blueprint_id, True)
+
+    @attr(client_min_version=base_test.LATEST_API_VERSION,
```

Comment:
client_min_version -> 3.1

Reply:
Changed to v2 for now. Waiting for Ido to answer.

Reply:
No need for a new class, using `client_min_version=2`